tensorflow_hub/tools/make_image_classifier/make_image_classifier_lib.py

```diff
@@ -191,7 +191,7 @@ def build_model(module_layer, hparams, image_size, num_classes):
   return model
 
 
-def train_model(model, hparams, train_data_and_size, valid_data_and_size):
+def train_model(model, hparams, train_data_and_size, valid_data_and_size, log_dir):
```

Comment:
Please provide a default value log_dir=None to not break existing callers of the library code.

Reply:
Thanks for the advice. Changed in the 00b6755 commit.